pyproject.toml

```diff
@@ -55,7 +55,7 @@ types-pyyaml = "^6.0.12.11"
 
 
 [tool.poetry.scripts]
-dsp-tools = "dsp_tools.dsp_tools:main"    # definition of the CLI entry point
+dsp-tools = "dsp_tools.cli_entry_point:main"    # definition of the CLI entry point
```

Comment:
see my note below

Reply:
and in terms of useability: how would you feel about the CLI command being `dsp` instead of `dsp-tools`? (for now you could even support both for a while, maybe with a deprecation warning)

Reply:
typing `dsp-tools ...` is probably quite deeply internalized in the users' brains, so I'm a bit reluctant of changing it. And also, the tool is called "DSP-TOOLS", and installed with `pip install dsp-tools`, so it is most natural to type `dsp-tools ...` in the Terminal.

I think that there must be a really strong reason to change something like this. Do you think that there is such a compelling reason? 

(I think the difference in the length is too small to justify such a change...)

Reply:
for your first point, I would argue: people can learn - I'm sure this change would be internalised rather quickly.  
for your second part, I would argue: no. 😉  there are many CLI tools that have a simpler entry point than their full /unique name (`brew` for Homebrew, `http` for HTTPie, etc.)

I personally think it would be nicer, not only because of it being shorter, but also because of it not needing the hyphen character (I'm most used to CLI tools being [a-z] only), and to me it would sound more like thought from the user perspective: they want to create a datamodel on the DSP, and do not necessarily care that they use DSP-TOOLS for that, so as a statement `dsp create` is closer to what they actually want than `dsp-tools create`.  
But no, I don't think there is an urgent reason for the change. So if you really deem it that radical, then it's not justified.